atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -126,29 +150,10 @@ static void waitForSchemaVersions(
         throw new IllegalStateException(errorMessage);
     }
 
-    static void runWithWaitingForSchemas(
-            RunnableCheckedException<TException> task,
-            CassandraKeyValueServiceConfig config,
-            CassandraClient client,
-            String unsafeSchemaChangeDescription)
-            throws TException {
-        waitForSchemaVersions(config, client, "before " + unsafeSchemaChangeDescription);
-        task.run();
-        waitForSchemaVersions(config, client, "after " + unsafeSchemaChangeDescription);
-    }
-
-    static boolean uniqueSchemaWithQuorumAgreementAndOtherNodesUnreachable(
-            CassandraKeyValueServiceConfig config,
+    private static boolean uniqueSchemaOnAllNodesAllowingForOneUnreachable(
```

Comment:
I'd prefer we left a comment explaining why we chose this, and why it's not a function of topology (following the discussion on the previous version of this PR)

Reply:
Added to the above java doc